test/plugins/compare-numbers-plugin-test.js

```
@@ -0,0 +1,43 @@
+const buster = require('buster')
+const plugin = require('../../lib/plugins/compare-numbers-plugin.js')
+
+buster.testCase('compare-numbers-plugin', {
+  'should throw an error if input is not a number': function () {
+    buster.assert.exception(() => plugin('asd', 123, { operator: '=' }))
+    buster.assert.exception(() => plugin(1, 'asd', { operator: '=' }))
+  },
+  'throw an exception if not known operator': function () {
```

should throw an exception

---

done